config/dev.sample.exs

```diff
@@ -9,3 +9,5 @@ config :at_ex,
   auth_token: "",
   username: "sandbox",
   endpoint: "sandbox"
```

Comment:
kindly remove the `endpoint` key as it serves the same purpose as the force live